docs/plugins.asciidoc

```diff
@@ -13,24 +13,18 @@ A list of existing Kibana plugins is available on https://github.com/elastic/kib
 Use the following command to install a plugin:
 
 [source,shell]
-bin/kibana plugin --install <org>/<package>/<version>
+bin/kibana-plugin install <package> (optional URL)
```

Comment:
This is a little ambiguous, so here are the three ways to install (using x-pack as an example).  You know longer ever specify both a name *and* url.

```
bin/kibana-plugin install x-pack
bin/kibana-plugin install https://example.com/x-pack.zip
bin/kibana-plugin install file:///some/local/path/x-pack.zip
```